builtin/ffi_overrides/init.lua

```diff
@@ -0,0 +1,66 @@
+local ie = ...
```

Comment:
IMO calling `request_insecure_environment()` where needed is better instead of centralizing the call.

Reply:
The reason for doing it this way was that `request_insecure_environment` can only be called from the `init.lua` (I think), but that could be relaxed for builtin in particular.